### ISSUE TITLE
Print dialog again

### ISSUE THREAD

**Issue by Charlotte-Morgan** [inasafe/inasafe#4803]
In our 4.3 walk through we had some shared concerns that the print dialog may be a bit wordy and confusing for users. we also discussed moving some text to help

Here are some suggestions to make it a little simpler. .. skip to the mock up below....
![image](https://user-images.githubusercontent.com/5743135/34090496-8d5ff58a-e40b-11e7-8416-ada73311bbd3.png)
main point is to give the user a choice of templates and only state once that the default will not be used if they choose one of the other options.

here is my mock up - with apologies to expert mocker uppers
![image](https://user-images.githubusercontent.com/5743135/34090570-27c29bd2-e40c-11e7-87fb-8a47d207370a.png)

The dots on the top should all be possible - ie can select all
the custom map should only be possible if the print map report option is not selected 
only one option should be possible in the custom map reports section

**Comment by myarjunar** [comment]
thanks for the feedback @Charlotte-Morgan , it looks much simpler than before. But what about putting the defaut map reports in the map report group? because the user should be able to open the default map template in qgis composer. so it would be something like this:
![impact report - preview _166](https://user-images.githubusercontent.com/11134669/34092722-746b8cba-e3f7-11e7-99ac-f9c6fbc6e237.png)

Or, we could just go with your mockup and simply put the `open in composer` button at the button group in the bottom.

**Comment by Gustry** [comment]
@Charlotte-Morgan You should update from develop, we made some updates last Thursday about wording in this dialog
